controllers/compassmanager_controller_test.go

```diff
@@ -6,8 +6,8 @@ import (
 
 	"github.com/kyma-project/compass-manager/api/v1beta1"
 	kyma "github.com/kyma-project/lifecycle-manager/api/v1beta2"
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
+	. "github.com/onsi/ginkgo/v2" //nolint:revive
```

Comment:
I don't understand what could be wrong with this line. Could you elaborate why it was failing and had to be ignored?

Reply:
It was failing because dot import. In most cases (e.g main files) that is bad practice to use this kind of import. But in Go test, if the programmer knows how dot imports work that could be allowed. 
In our case, we can remove `.` and replace it with any name we wan. But after that we need to add that name before any function call exported from that package (e.g Describe(), It() ). Such a way will make the main loops of the program less readable and transparent. So that is why I chose that approach.